gpu_ocean/SWESimulators/SimWriter.py

```diff
@@ -166,10 +166,18 @@ def __init__(self, sim, num_layers=1, staggered_grid=False, \
             self.ncfile.createDimension('x', nx)
             self.ncfile.createDimension('y', ny)
         if (not self.ignore_ghostcells) and (self.staggered_grid):
-            self.ncfile.createDimension('x_hu',   nx + self.ghost_cells_tot_x + 1)
-            self.ncfile.createDimension('y_hu',   ny + self.ghost_cells_tot_y)
-            self.ncfile.createDimension('x_hv',   nx + self.ghost_cells_tot_x)
-            self.ncfile.createDimension('y_hv',   ny + self.ghost_cells_tot_y + 1)
+            if (fbl):
```

Comment:
Perhaps it would be better to use the variable `self.simulator_short` instead of introducing a new input parameter?

I think that the only required comment here would be "Create dimension according to data sizes required by the FBL simulator". If the reader wonders what that means, the reader should look into the FBL class, or in our slides. 

I think it is important not to duplicate advanced comments unnecessarily, as such comments are easy to forget to modify during a refactoring

Reply:
Definitely!